### ISSUE TITLE
How to run the toc generator?

### ISSUE THREAD

**Issue by misappi** [todogroup/outbound-oss#73]
Hi @cornelius,
How do you normally run the toc generator? Manually on your local machine?  
Would it make sense to run it automatically via GH action - maybe after each push?

**Comment by cornelius** [comment]
Yes, I have ran it manually so far. But I agree, it would make sense to automate it with GitHub Action.

**Comment by misappi** [comment]
@cornelius : OK, I'll change this issue accordingly.

**Comment by cornelius** [comment]
If we run a GitHub Action, we could also generate a one page version of the guide by concatenating the source files and adjusting the relative links. This would make it easier to sync with the version on the TODO group web site or the PDF version.

**Comment by misappi** [comment]
> If we run a GitHub Action, we could also generate a one page version of the guide by concatenating the source files and adjusting the relative links. This would make it easier to sync with the version on the TODO group web site or the PDF version.

Makes sense. Should we create a separate issue for that?
